src/Component/Reactdltrow.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react'
+import ReactDeleteRow from 'react-delete-row';
+
+
+export default class TodoApi extends PureComponent {
+    
+  render() {
+    const items = this.props.todos.todos;
```

Comment:
remove console and no need to create variable in render use directly using && operator